### ISSUE TITLE
Can't compile with cmake 3.11.3

### ISSUE THREAD

**Issue by harryklein** [openambitproject/openambit#193]
I try to compile openambit with cmake 3.11.3 (archlinux). I get the following compile error:

```
------building movescount------
-- The CXX compiler identification is GNU 8.1.1
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found ZLIB: /usr/lib/libz.so (found version "1.2.11") 
-- Found libambit: /home/harald/project/openambit.orig/openambit/src/libambit, /home/harald/project/openambit.orig/openambit/libambit-build/libambit.so
CMake Error at CMakeLists.txt:62 (QT5_USE_MODULES):
  Unknown CMake command "QT5_USE_MODULES".


-- Configuring incomplete, errors occurred!
See also "/home/harald/project/openambit.orig/openambit/movescount-build/CMakeFiles/CMakeOutput.log".
make: *** Es wurden keine Ziele angegeben und keine „make“-Steuerdatei gefunden.  Schluss.
------building openambit------
-- The CXX compiler identification is GNU 8.1.1
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found libambit: /home/harald/project/openambit.orig/openambit/src/libambit, /home/harald/project/openambit.orig/openambit/libambit-build/libambit.so
-- Movescount not found, building from source in /home/harald/project/openambit.orig/openambit/src/openambit/../
-- Found UDev: /usr/include  
CMake Error at CMakeLists.txt:103 (QT5_USE_MODULES):
  Unknown CMake command "QT5_USE_MODULES".


-- Configuring incomplete, errors occurred!
See also "/home/harald/project/openambit.orig/openambit/openambit-build/CMakeFiles/CMakeOutput.log".
make: *** Es wurden keine Ziele angegeben und keine „make“-Steuerdatei gefunden.  Schluss.
```

**Comment by aicastell** [comment]
Try installing qtbase5-dev package. On Debian/Ubuntu based distros:

    $ sudo apt-get install qtbase5-dev

(Search for the proper way to do that on Arch Linux)

After that, repeat the compilation process.

**Comment by mbernasocchi** [comment]
@harryklein does this work now?

**Comment by mbernasocchi** [comment]
closing as it seems fixed in 92c69a40be7903007d1e4e3ea542d2000a77046c
reopen if necessary
